python/client.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import time
+import pytest
+
+retry_timeout = 3
+limit = 4
+good_version_couter = 0
+good_version = "v0.4.21"
+retry_number = 12
+
+class RestfulOneQ:
+
+    _s = requests.session()
+    host = None
+
+    def __init__(self, host):
+        self.host = host
+
+
+    def create_task(self, data: dict):
+        return self._s.post(self.host + "/task", json=data)
+
+    def get_task_by_id(self, id):
+        return self._s.get(self.host + "/task/" + id)
+
+
+    def send_version(self, retry_count):
+        global good_version_couter
+        response = self._s.get(self.host + "/version")
+        # self.retry_count = retry_count
+
+        print("Version from response - " + response.json())
+
+        if(response.json() == good_version):
+            print("good_version_couter before increment = " + str(good_version_couter))
```

Comment:
Вместо сложения строк можно писать через запятую:
```suggestion
            print("good_version_couter before increment = ", good_version_couter)
```